LibertyDSNP/activity-content
# update to node 16

Problem
=======
Updating Nodejs from 14 to 16
#9 

Solution
========
I updated the proper config and tool versions from node engine from 14.xx.xx to 16.14.0

Change summary:
---------------
* Changed the asdf install to node 16
* Updated tool-version, .node-version, github actions, package json
* Removed 'npm update v7' github action

Steps to Verify:
----------------
1. run: asdf install node 16
2. run: asdf which node (ensure the install is node 16.14.0
3. go to example-client
4. npm install @dsnp/activity-content
5. compile and run example-client
